tests/model_bad_rel1.py

```diff
@@ -7,7 +7,10 @@
 
 class Tournament(Model):
     id = fields.IntField(pk=True)
+    events: fields.RelationQueryContainer["Event"]
```

Comment:
We really needn't change these `model_*` models as they are faulty on purpose for testing.

Reply:
No problems, I will return them to their original state except for the models from `testmodels`.